src/features/agreements-accordion/agreements-accordion.js

```diff
@@ -83,7 +83,7 @@ const AgreementsAccordion = ({
     if (stripes.hasPerm('erm.agreements.collection.get')) {
       getAgreements(refId);
     }
-  }, []);
+  }, [getAgreements, refId, stripes]);
```

Comment:
This acts as a didMount method, so deps array should stay empty. Please add a `// eslint-disable-next-line react/exhaustive-deps`

Reply:
Done.